capstone/src/main/java/com/google/sps/servlets/Login.java

```diff
@@ -22,8 +22,8 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       logMsg = "user is now logged in";
       redirectUrl = userService.createLoginURL("/login");
     } else {
-      logMsg = "user is now logged out";
-      redirectUrl = userService.createLogoutURL("/");
+      logMsg = "user is already logged in";
```

Comment:
Was the earlier bug with authentication fixed?

Reply:
I am not sure, authentication is not my feature

Reply:
I took out line 26 and rerouted to the stats page. This is a fix to the logout bug, but temporarily no one can log out.